package.json

```diff
@@ -21,7 +21,7 @@
     "enzyme-adapter-react-16": "^1.0.5",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-15": "^1.0.5",
-    "eslint": "^4.17.0",
+    "eslint": "4.15.0",
```

Comment:
reason for this?

Reply:
`react-scripts@latest` gets really picky about the versions of `eslint` & `jest` (hence also why jest is only updated to `22.1` instead of `22.4`).

Reply:
It's not awesome, but seeing as it's a minor version I think it might be OK. Something to keep on the radar though - `react-scripts` is a bit of a leech package at the moment.